apisix/plugins/openid-connect.lua

```diff
@@ -65,7 +65,7 @@ local schema = {
         },
         redirect_uri = {
             type = "string",
-            description = "use ngx.var.request_uri if not configured"
+            description = "auto append '.apisix/redirect' if not configured"
```

Comment:
But it cannot make sure the new URI will be matched by the same route, it actually adds an implicit condition that the route match type is prefix instead of exact.

Reply:
Well. If the route is of type **exact**, then it cannot actually use this `openid-connect` plugin, because the `redirect_uri` must be different from the request URI.

Reply:
@liweitianux Could you write some hints about this in the OpenID Connect doc?

Reply:
Sure. I'll update the doc later.